docs/src/main/sphinx/functions/session.rst

```diff
@@ -8,4 +8,12 @@ Functions providing information about the query execution environment.
 
     Returns the current user running the query.
 
+.. data:: current_catalog
+
+    Returns a character string that represents the current default catalog name.
+
+.. data:: current_schema
+
+    Returns a character string that represents the current default unqualified schema name.
```

Comment:
Remove the word "default"

Reply:
Removed "default" from the definition of current_schema parameter.